service/plugin/src/main/java/com/solace/maas/ep/runtime/agent/plugin/processor/logging/MDCProcessor.java

```diff
@@ -0,0 +1,22 @@
+package com.solace.maas.ep.runtime.agent.plugin.processor.logging;
+
+import com.solace.maas.ep.runtime.agent.plugin.constants.RouteConstants;
+import org.apache.camel.Exchange;
+import org.apache.camel.Processor;
+import org.slf4j.MDC;
+import org.springframework.stereotype.Component;
+
+@Component
+public class MDCProcessor implements Processor {
+    @Override
+    public void process(Exchange exchange) throws Exception {
```

Comment:
Isn't ScanType missing here?

Reply:
@slunelsolace, Thanks for the follow up, I have updated the interceptor to set the scanType in the header and updated the MDCProcessor accordingly.